Jellyfin.Plugin.Template/Jellyfin.Plugin.Template.csproj

```diff
@@ -11,8 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
-    <PackageReference Include="Jellyfin.Model" Version="10.*-*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.8.0" />
+    <PackageReference Include="Jellyfin.Model" Version="10.8.0" />
```

Comment:
Only reason it was set to floating latest is because of the nightly builds, but I think we discontinued nightly plugin builds a while back?

I've mentioned wanting to set them static several times, and was given the go-ahead iirc, just never actually did anything about it.

I *could* make JPRM use the targetAbi, or add an additional parameter for setting the Jellyfin.* package versions.

I would at least want the template repo to have the version explicitly set, as that's what is used to test JPRM CI.

@joshuaboniface I think you gave the thumbs up last time I was talking about this, got any input on this?

Reply:
>Only reason it was set to floating latest is because of the nightly builds, but I think we discontinued nightly plugin builds a while back?

Yep, nightlies are no longer a thing at all. Might do weeklies in the future but yea.

I'd be OK with statics if there was a way to "not have to think about it", which the integrated CI could do, so, my input is let's try it and see.

Reply:
I am :100: in favour of pining the versions, just remembered we had it for some explicit reason and could not remember why we had it in the first place.
Thank you for the clarification!

Reply:
I have a green checkmate but cant trigger the merge, so whoever has the perms, my comment was addressed :)

Reply:
Yes, with this PR, dependabot *should* kick in on Jellyfin releases too.

I think this repo might be core only? I don't have merge rights either.

Reply:
*renovate-bot no dependabot no more :stuck_out_tongue_closed_eyes:

Reply:
*looks at open PRs*
If you say so :smile: